optaweb-employee-rostering-backend/src/main/java/org/optaweb/employeerostering/service/roster/RosterController.java

```diff
@@ -108,6 +120,36 @@ public RosterController(RosterService rosterService) {
                                                                         spots), HttpStatus.OK);
     }
 
+    @ApiOperation("Get a shift roster view between two dates for a subset of the spots as an excel file")
+    @GetMapping(value = "/shiftRosterView/excel",
+            produces = "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet")
+    public ResponseEntity<byte[]> getShiftRosterViewAsExcel(@PathVariable @Min(0) Integer tenantId,
+                                                            @RequestParam(name = "startDate")
+                                                                    String startDateString,
+                                                            @RequestParam(name = "endDate") String endDateString,
+                                                            @RequestParam(name = "spotList") String spotListString) {
+        Set<Long> spotIdSet = Arrays.stream(spotListString.split(",")).map(Long::parseLong)
```

Comment:
If the list contains a bad value, we get an uncaught exception. The try/catch below should probably cover this as well. 
The `RosterControllerTest` should cover this:
- wrong spot ID list => error response entity
- good spot ID list => call roster service with the same list of IDs